src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const stringToObject = sourceString
+    .split(';')
+    .map((value) => {
+      return value.split(':').map((spaces) => spaces.trim());
+    })
+    .reduce((acc, key) => {
+      acc[key[0]] = key[1];
+
+      return acc;
+    }, {});
```

Comment:
The chained methods should have the same indentation to improve readability. Align the '.map' and '.reduce' methods with the '.split' method.